src/tooling/piral-cli/src/injectors/pilet.ts

```diff
@@ -85,14 +94,43 @@ export default class PiletInjector implements KrasInjector {
     };
   }
 
-  getMeta() {
-    const { pilets } = this.config;
+  async getMeta() {
+    const { pilets, feed } = this.config;
 
-    if (pilets.length === 1) {
-      return JSON.stringify(this.getMetaOf(0));
+    const localPilets = pilets.map((_, i) => this.getMetaOf(i));
+    const mergedPilets = this.mergePilets(localPilets, await this.loadRemoteFeed(feed));
```

Comment:
Here feed is `string | undefined`, but this is not reflected in the parameters of `loadRemoteFeed`. 

Easiest option is below at `loadRemoteFeed`:

```ts
async loadRemoteFeed(feed?: string) {
  if (!feed) {
    return undefined;
  }

  // ... rest as-is
}
```

Reply:
oh yes, that got lost in the refactoring
added the check again